.github/workflows/release.yml

```diff
@@ -0,0 +1,293 @@
+# This file was autogenerated by dist: https://opensource.axo.dev/cargo-dist/
```

Comment:
When the `dist-workspace.toml` changes, either via manual edits or by `dist init`, running `dist generate` will rewrite this workflow.

There are CI checks in place to catch uncommitted diffs, like we do for our openapi spec, etc.